requirements.txt

```diff
@@ -9,3 +9,5 @@ attrs==19.3.0
 Flask-Commonmark==0.8
 bcrypt==3.2.0
 boto3==1.16.50
+pandas==1.2.4
```

Comment:
Hi @TiBiBa ! Could we remove this dependency?

Reply:
Completely forgot about that one, will remove it right away!